.github/workflows/buf-ci.yaml

```diff
@@ -26,4 +26,5 @@ jobs:
           username: ${{ secrets.BUF_USER }}
           token: ${{ secrets.BUF_TOKEN }}
           push_create_visibility: public
+          push_git_metadata: false
```

Comment:
Control of `push_git_metadata` is being removed from `buf-action`. This will break in the next version.